tests/test_flake8_trio.py

```diff
@@ -351,14 +356,15 @@ def info_tuple(error: Error):
 
     for err, exp in zip(errors, expected):
         err_msg = info_tuple(err)
-        for err, type_ in zip(err_msg, (int, int, str, type)):
+        for err, type_ in zip(err_msg, (int, int, str, type(None))):
```

Comment:
This fourth parameter is a relic, and flake8 explicitly don't care about it anymore. During refactoring I had trouble returning the type of the plugin and started returning `type(None)` - which required this minor change in the tests.